src/hotspot/share/gc/shared/collectedHeap.cpp

```diff
@@ -446,6 +446,12 @@ CollectedHeap::fill_with_array(HeapWord* start, size_t words, bool zap)
   ObjArrayAllocator allocator(Universe::intArrayKlassObj(), words, (int)len, /* do_zero */ false);
   allocator.initialize(start);
   DEBUG_ONLY(zap_filler_array(start, words, zap);)
+  if (DumpSharedSpaces) {
```

Comment:
Probably shouldn't both zap and clear for dumping, to avoid wasting time.

Reply:
Fixed.